uint8 robot_id
---
geometry_msgs/Point new_goal